### PR TITLE
feat: support custom token type header

### PR DESCRIPTION
**Pull request by dev-marek**
**Issue**

https://gravitee.atlassian.net/browse/APIM-7273

**Description**

image

**Additional context**

---
**Gravitee.io Automatic Deployment**

🚀 A prerelease version of this package has been published on Gravitee's private artifactory, you can:
 - use it directly by updating your project with version: `5.2.0-apim-7273-accept-at-jwt-token-typ-SNAPSHOT`
 - download it from Artifactory [here](https://odbxikk7vo-artifactory.services.clever-cloud.com/gravitee-snapshots/io/gravitee/policy/gravitee-policy-jwt/5.2.0-apim-7273-accept-at-jwt-token-typ-SNAPSHOT/gravitee-policy-jwt-5.2.0-apim-7273-accept-at-jwt-token-typ-SNAPSHOT.zip)

### CODE REVIEW

File: src/test/java/io/gravitee/policy/jwt/JWTPolicyTest.java [hunk 1]
```diff
@@ -394,6 +393,54 @@ void extractSecurityToken_shouldReturnEmpty_whenTokenIsAbsent() {
...
+        when(jwtProcessor.process(any(JWT.class), isNull())).thenReturn(claimsSet);
+        when(ctx.request()).thenReturn(request);
+        when(ctx.metrics()).thenReturn(metrics);
+        when(request.headers()).thenReturn(headers);
+
+        final TestObserver<Void> obs = cut.onRequest(ctx).test();
+        obs.assertComplete();
+
+        verifyMetricsAttributesAndHeaders(metrics, headers, CLIENT_ID, STANDARD_SUBJECT);
+    }
```
[the remark this answers is not in the document]
Comment (dev-marek) [reply src/test/java/io/gravitee/policy/jwt/JWTPolicyTest.java]: Yes, that was a placeholder for a proper test I didn't removed. Thanks for pointing this out!

**Review by illichso — APPROVED** [review]

**Pull request merged**